composer.json

```diff
@@ -37,6 +37,7 @@
         "mongodb/laravel-mongodb": "^5.0",
         "orchestra/testbench": "^8.0|^9.0|^10.0",
         "orchestra/testbench-core": "^8.0|^9.0|^10.0",
+        "orchestra/workbench": "^8.0|^9.0|^10.0",
```

Comment:
We use workbench so having it as a dependency makes sense. This also fixed an issue where we were getting a broken version of `orchestra/canvas`.